### ISSUE TITLE
Tiles are getting unavailable and not working after sometimes and sometimes working fine

### ISSUE THREAD

**Issue by Priyanshu-apk** [flxholle/QuickTiles#12]
Can you please resolve this. I am using Android 14 almost stock version on Motorola Phone
And thanks for this App this App done the work of 4 - 5 apps in 1 ...I uninstalled 3 apps after installing this all in one app but sometimes tiles becomes unavailable. I have turned off battery optimisation and lifted off this app from any restrictions but still I am facing this issue

**Comment by Priyanshu-apk** [comment]
![Screenshot_20241116-071053~2](https://github.com/user-attachments/assets/49851adf-0c17-45c1-955d-99bce93d842f)

**Comment by Anujkaushik01** [comment]
Only unavailable is written but the tile still works when you tap on it. Like I use split screen tile and it become unavailble but when I click it, app goes into split screen.

**Comment by Priyanshu-apk** [comment]
Yes sometimes Unavailable tiles still works after like screen timeout tile starts working after clicking 

Some of them like volume panel don't work after clicking if it is unavailable 

Some of them don't work at all no matter they are available or not, and suddenly starts working after few hours

Also I noticed that this issue is not only in quicktiles but also in other such tile apps which are not from play store...is there anything in android which causes this behaviour.

**Comment by flxholle** [comment]
#26 should fix this
